src/det.jl

```diff
@@ -21,6 +21,23 @@ end
     return vecdot(x0, cross(x1, x2))
 end
 
+@inline function _det(::Size{(4,4)}, A::StaticMatrix,S::Type)
+    A = similar_type(A,S)(A)
```

Comment:
Any preferred style?

I though of changing each function that this line appears as the first line in this way:

```
@inline function _det(::Size{(3,3)}, A::StaticMatrix, S::Type)
    A = similar_type(A,S)(A)
```

would become:

```
@inline function _det(::Size{(3,3)}, _A::StaticMatrix, S::Type)
    A = similar_type(_A,S)(_A)
```